### PR TITLE
Adding Volume-of-sphere Exercise

### PR DESCRIPTION
**Pull request by FrancisJ000**
Added an exercise to find the volume of a sphere.

### CODE REVIEW

File: exercises/volume of sphere/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+To find the volume of a sphere
+
+For example:
+input: "1"
+output: "4.1887902047863905"
+
+## Source
+
+The official Julia doc https://juliabyexample.helpmanual.io/
```
Comment (logankilpatrick) [inline exercises/volume of sphere/README.md]: This is not the official Julia docs.

Comment (logankilpatrick) [reply exercises/volume of sphere/README.md]: These are: https://docs.julialang.org/en/v1/

File: exercises/volume of sphere/volume.jl [hunk 2]
```diff
@@ -0,0 +1,4 @@
+function sphere_vol(r)
```
Comment (logankilpatrick) [inline exercises/volume of sphere/volume.jl]: Why is this function blank?

**Comment by SaschaMann** [comment]
Good call. It's also a bit too simple for an exercise, there's not really anything to discuss in its current state.

**Pull request closed without merging**